examples/storage/minio/README.md

```diff
@@ -0,0 +1,196 @@
+# Cloud Native Deployment of Minio using Kubernetes
```

Comment:
Since `Minio` and `distributed Minio` deployments use different constructs (deployment and statefulset respectively), it will be difficult to convey info clearly if both docs are combined. What do you think?